ui/cypress/integration/common-features/user_login_spec.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 PingCAP, Inc. Licensed under Apache-2.0.
+
+// Test User login
+describe('Root User Login', () => {
+  beforeEach(function () {
+    cy.fixture('uri.json').as('uri')
+    cy.visit('@uri.root')
+  })
+
+  it('authenticated redirect', function () {
+    // Redirect to login
+    cy.visit(this.uri.root)
```

Comment:
Can these two uses be the same style? For example, both use `@uri.root` or `this.uri.root`. I don't know if this is possible.

Reply:
Both `@uri.root` and `this.uri.root` work fine, but you are right, we'd better use one style. I chose `this.uri.root`, since it will be easier to use in assertion.